src/Context/Extensions/ProfileLevel.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace MongoDB.Extensions.Context
+{
+    public enum ProfileLevel
```

Comment:
I guess this is a fixed enum, otherwise smartenums could come handy. just a choice/preference in tastes though...

Reply:
I think a fixed enum is okay here...